### PR TITLE
Ethbridge 0.39 Genesis Import/Export

### PR DESCRIPTION
**Pull request by AustinoBombino**

### CODE REVIEW

File: tools/sifgen/genesis/genesis.go [hunk 1]
```diff
@@ -144,6 +145,44 @@ func InitializeCLP(nodeHomeDir, clpConfigURL string) error {
 	return nil
 }
 
+func InitializeEthbridge(nodeHomeDir, ethbridgeConfigURL string) error {
+	genesis, err := readGenesis(nodeHomeDir)
+	if err != nil {
+		return err
+	}
+
+	response, err := http.Get(fmt.Sprintf("%v", ethbridgeConfigURL))
```
Comment (banshee) [inline tools/sifgen/genesis/genesis.go:154]: We shouldn't be connecting to the network to get this.

**Review by banshee — CHANGES_REQUESTED** [review]

**Review by intl-man — CHANGES_REQUESTED** [review]
@AustinoBombino Can you explain more of the intention behind this? Sifgen was only ever designed to be something that could allow us to spin up new networks/nodes quickly.

**Comment by AustinoBombino** [comment]
> @AustinoBombino Can you explain more of the intention behind this? Sifgen was only ever designed to be something that could allow us to spin up new networks/nodes quickly.

@intl-man The integration tests were failing prior to adding the new genesis info to sifgen, so I added it in an attempt to fix that.
But the `InitializeEthbridge()` function was just an attempt to match the existing CLP logic. If you're saying that it isn't necessary we can definitely remove it. 

Can you explain the purpose of the existing `InitializeCLP()` function?

**Comment by AustinoBombino** [comment]
@intl-man @banshee 
I tried removing the `InitializeEthbridge()` function and ran the integration tests locally. Getting failures on these 8:

> src/py/test_rowan_transfers.py::test_rowan_to_erowan FAILED                                                                                                                                                       
> src/py/test_random_currency_roundtrip.py::test_three_letter_currency_with_capitals_in_name FAILED 
> src/py/test_random_currency_roundtrip.py::test_transfer_tokens_with_some_currency FAILED
> src/py/test_peggy_fees.py::test_fee_charged_to_transfer_rowan_to_erowan FAILED
> src/py/test_peggy_fees.py::test_ceth_receiver_account FAILED  
> src/py/test_peggy_fees.py::test_rescue_ceth FAILED     
> src/py/test_eth_transfers.py::test_eth_to_ceth_and_back_to_eth FAILED
> src/py/test_ebrelayer_restart.py::test_sifchain_transactions_with_offline_relayer FAILED

File: tools/sifgen/genesis/genesis.go (hunk 1 above)
Comment (intl-man) [reply tools/sifgen/genesis/genesis.go:154]: > > @AustinoBombino Can you explain more of the intention behind this? Sifgen was only ever designed to be something that could allow us to spin up new networks/nodes quickly.
> 
> @intl-man The integration tests were failing prior to adding the new genesis info to sifgen, so I added it in an attempt to fix that.
> But the `InitializeEthbridge()` function was just an attempt to match the existing CLP logic. If you're saying that it isn't necessary we can definitely remove it.
> 
> Can you explain the purpose of the existing `InitializeCLP()` function?

So, that was just a temp solution when we needed to import existing pools when testing the `changePoolFormula` release, back in March. It's no longer needed and should be removed.

**Comment by intl-man** [comment]
> @intl-man @banshee
> I tried removing the `InitializeEthbridge()` function and ran the integration tests locally. Getting failures on these 8:
> 
> > src/py/test_rowan_transfers.py::test_rowan_to_erowan FAILED
> > src/py/test_random_currency_roundtrip.py::test_three_letter_currency_with_capitals_in_name FAILED
> > src/py/test_random_currency_roundtrip.py::test_transfer_tokens_with_some_currency FAILED
> > src/py/test_peggy_fees.py::test_fee_charged_to_transfer_rowan_to_erowan FAILED
> > src/py/test_peggy_fees.py::test_ceth_receiver_account FAILED
> > src/py/test_peggy_fees.py::test_rescue_ceth FAILED
> > src/py/test_eth_transfers.py::test_eth_to_ceth_and_back_to_eth FAILED
> > src/py/test_ebrelayer_restart.py::test_sifchain_transactions_with_offline_relayer FAILED

What should the EthBridge be initialized with? Currently it's just `null`.

**Review by timlind — APPROVED** [review]

**Comment by intl-man** [comment]
@AustinoBombino Closing as this is no longer applicable with 0.42.

**Pull request closed without merging**